frontend/src/budget_simulation/BudgetSimulation.js

```diff
@@ -21,15 +18,75 @@ const resources = [
     "education",
 ];
 
+/**
+ * Component for displaying citizen information
+ *
+ * The fill indicates the citizen's stance on the budget,
+ * and hovering over the component displays the citizen's traits
+ */
+
+class Citizen extends React.Component {
+    constructor(props){
+        super(props);
+        this.state = {
+            show: false,
+        };
+    }
+
+    render() {
+        const statistic = (
+            <Popover id={"popover-basic"}>
+                <Popover.Title as={"h3"}> {this.props.data.name} </Popover.Title>
+                <Popover.Content>
+                    {Object.keys(this.props.data.traits).map((trait, key) =>
+                        (<div key={key}>
+                            <strong>
+                                {trait.split("_").join(" ")}: &nbsp;
+                            </strong>
+                            {this.props.data.traits[trait] ? "True" : "False"}
+                            <br/>
+                        </div>)
+                    )}
+                </Popover.Content>
+            </Popover>
+        );
+        return (
+            <OverlayTrigger
+                overlay={statistic}
+                placement={"right"}
+            >
+                <svg className={"budget-reaction-citizen"} height="20" width="20">
+                    <circle
+                        cx="10"
+                        cy="10"
+                        r="10"
+                        fill={this.props.data["will_support"] ? "green" : "#c0c0c0"}
```

Comment:
instead of setting raw hex codes, better to assign this a name (so, e.g., `const grey_hex = '#c0c0c0'` and then here:

`fill={this.props.data["will_support"] ? "green" : grey_hex }`

... will make this more maintainable.